### PR TITLE
Add parameters to navigational suggestions merino job

### PR DESCRIPTION
**Pull request by abhi-agg**
Required to fix:
https://mozilla-hub.atlassian.net/browse/DENG-834
https://mozilla-hub.atlassian.net/browse/DENG-833

### CODE REVIEW

File: dags/merino_jobs.py [hunk 1]
```diff
@@ -142,6 +142,9 @@ def merino_job(
             "moz-fx-merino-nonprod-ee93",
             "--dst-gcs-bucket",
             "merino-images-stagepy",
+            "--dst-cdn-hostname",
+            "stagepy-images.merino.nonprod.cloudops.mozgcp.net"
```
Comment (quiiver) [inline dags/merino_jobs.py:146]: need commas after all arguments

**Review by quiiver — APPROVED** [review]
rwc+

File: dags/merino_jobs.py (hunk 1 above)
Comment (abhi-agg) [reply dags/merino_jobs.py:146]: Ahh shoot. Forgot it.

**Review by ncloudioj — APPROVED** [review]
:shipit:

**Pull request merged**